src/SumoCoders/FrameworkCoreBundle/Tests/Controller/DatePickerControllerTest.php

```diff
@@ -122,22 +82,14 @@ public function testIfSingleTextWithDatePickerAndOnlyDatesInThePastRenderedCorre
         $endDate = new \DateTime('next friday');
 
         $element = $crawler->filter('input#form_date_example7');
```

Comment:
don't forget to adjust this id when shifting the ids in the code

Reply:
fixed 89a0a2e